src/luvi_compat.c

```diff
@@ -27,8 +34,16 @@ void luvi_openlibs(lua_State *L) {
 
     luaL_register(L, LUA_STRLIBNAME, funcs);
   }
+  lua_pop(L, 1);
 
   luaL_requiref(L, LUA_UTF8LIBNAME, luaopen_utf8, 1);
+
+#if (LUA_VERSION_NUM == 501)
+
+  lua_pushlstring(L, UTF8PATT_501, sizeof(UTF8PATT_501)/sizeof(char) - 1);
+  lua_setfield(L, -2, "charpattern");
   lua_pop(L, 1);
```

Comment:
This pop should be outside of the `#if`, since its popping the value from `luaL_requiref`, right?

Reply:
Yep.